### ISSUE TITLE
Wasted Youth, Part 1: very loud audio glitch when running

### ISSUE THREAD

**Issue by hedgehog90** [ruffle-rs/ruffle#10523]
### Describe the bug

Sample SWF:
https://github.com/hedgehog90/WYP1/blob/main/wy.swf

Sample video (glitch appears at around 0:18 and 1:07):
https://www.youtube.com/watch?v=yadw1J_v9lo

Not sure what triggers it. Many audio events happening at once (footsteps) and they are panned left and right depending how far from the centre of the screen you are. I can trigger massive lag by moving the mouse, which also appears to help trigger it. As you can see from the video though it happens occasionally but not all the time.

### Expected behavior

Normal audio.

### Affected platform

Desktop app

### Operating system

Windows 11

### Additional information

ruffle-nightly-2023_04_02-windows-x86_64

**Comment by Lord-McSweeney** [comment]
The audio problems are probably caused by lag.

**Comment by n0samu** [comment]
That shouldn't be happening though! It seems like a similar issue to #8695. @hedgehog90 can you check if the problem also happens in the browser? (For example, by playing the game in the Ruffle Web Demo.) The problem in HitBox only happens in the desktop app so I'm curious if it's the same for this game.

**Comment by hedgehog90** [comment]
Yes, it's evident in the web demo. Interestingly the mouse lag is completely gone in the browser version, so the audio issue is not due to lag.

**Comment by n0samu** [comment]
Thanks for the additional information, and I'm glad you got the issue resolved!

**Comment by hedgehog90** [comment]
Just to clarify, the issue with the sudden explosions of noise in Ruffle hasn't been resolved and needs to be looked into.

**Comment by n0samu** [comment]
Hmm, maybe this is actually the same problem as #9763?

**Comment by hedgehog90** [comment]
I think so. I suspect it might be something to do with sounds that are panned left/right.

**Comment by hedgehog90** [comment]
I think #10523 is related, also an issue with panning I think.

**Comment by hedgehog90** [comment]
I've just tested  again with the latest web build to check if this issue was resolved. Perhaps because I tested a different scenario, but it appeared to be much improved as I only noticed the audio glitch once.
As stated above, I believe the audio glitch appears when there are multiple sounds play simultaneously with different panning values. Would appreciate if someone could look into this.
